community/import-tool/src/main/java/org/neo4j/tooling/ImportTool.java

```diff
@@ -443,7 +443,7 @@ public static void main( String[] incomingArguments, boolean defaultSettingsSuit
             input = new CsvInput( nodeData( inputEncoding, nodesFiles ), defaultFormatNodeFileHeader(),
                     relationshipData( inputEncoding, relationshipsFiles ), defaultFormatRelationshipFileHeader(),
                     idType, csvConfiguration( args, defaultSettingsSuitableForTests ), badCollector,
-                    configuration.maxNumberOfProcessors() );
+                    configuration.maxNumberOfProcessors(), !skipBadRelationships );
```

Comment:
Hmm, well... In each of the contexts I prefer them as they are. Otherwise in `CsvInput` that would look like `skipRelationshipValidation` which feels a little reversed, where the check in the implementation would be `if ( !skipRelationshipValidation ) doValidation();`